### PR TITLE
2 create mvp

### PR DESCRIPTION
**Pull request by jah71**

### CODE REVIEW

File: app/src/App.js [hunk 1]
```diff
@@ -1,23 +1,56 @@
...
 
 function App() {
+  const [searchValue, setSearchValue] = useState('');
+  const [matches, setMatches] = useState([]);
+
+  async function search(e) {
+    e.preventDefault();
+    
+    const matchData = await getSummonerByName(searchValue);
+    const dataJson = await matchData.json();
```
Comment (jah71) [inline app/src/App.js:13]: could we name this something more descriptive

File: app/src/App.js [hunk 2]
```diff
@@ -1,23 +1,56 @@
...
-            <button className="btn-summoner-name-search">Search</button>
-          </div>
-          <div>
-            TODO: Table goes here
-          </div>
+    <div className="main-content">
+      <form name="get-stats" onSubmit={search}>
+        <div className="search-bar">
+          <label>Search </label>
+          <input name="search" placeholder="Summoner name" onChange={e => setSearchValue(e.target.value)} data-testid="inp-summoner-serach" />
```
Comment (jah71) [inline app/src/App.js:22]: no aria-label here, so screen readers might get confused here

File: app/src/App.js [hunk 3]
```diff
@@ -1,23 +1,56 @@
...
+              <th colSpan="1">Result</th>
+              <th colSpan="1">Champion</th>
+              <th colSpan="1">K/D/A</th>
+            </tr>
+          </thead>
+          <tbody>
+            {matches.map((match, index) => (
+              <tr data-testid={'table-row-' + index} className="table-row" key={index}>
+                <td className="table-col">{match}</td>
+                <td className="table-col">Zac</td>
```
Comment (jah71) [inline app/src/App.js:42]: hard coded value here, america explain

**Review by jah71 — COMMENTED** [review]
it's probably fine for now

**Review by zacbryantdev — APPROVED** [review]

**Pull request merged**